test/helpers/contract-helpers.ts

```diff
@@ -17,7 +18,7 @@ const SHARES_RSA = [
 ];
 
 export const DataGenerator = {
```

Comment:
Remove SIGNATURE, SHARES_RSA, DataGenerator.publicKey, and DataGenerator.shares
With your new function, this is no longer necessary.